index.js

```diff
@@ -189,3 +169,47 @@ async function invalidation (invalidate, cache, name, self, arg, ctx, info, resu
     onError(err)
   }
 }
+
+async function getResultForMutationSubscription ({ self, arg, ctx, info, originalFieldResolver, onError }) {
+  let result = false
+  if (info.operation && (info.operation.operation === 'mutation' || info.operation.operation === 'subscription')) {
+    try {
+      result = await originalFieldResolver(self, arg, ctx, info)
+    } catch (error) {
+      onError(error)
+      return null
```

Comment:
This is not correct and a bug in the previous implementation. We should not catch this error nor invoke onError.